tests/cases/fourslash/goToDefinitionSatisfiesExpression1.ts

```diff
@@ -0,0 +1,14 @@
+/// <reference path="./fourslash.ts"/>
+
+// @noImplicitOverride: true

```

Comment:
What is this doing?

Reply:
😅 It's copied from other files accidentally.